lib/stitch_plan/stitch.py

```diff
@@ -91,3 +91,7 @@ def __json__(self):
         attributes = dict(vars(self))
         attributes['tags'] = list(attributes['tags'])
         return attributes
+
+    def as_point(self) -> Point:
```

Comment:
What's the goal here?  Stitch inherits from Point, so a Stitch already is a Point.

Reply:
Good call, missed the inheritance.